modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js

```diff
@@ -0,0 +1,260 @@
+        $("#smoother-" + type).html("");
+        $("#legend-" + type).html("");
+        $("#chart-" + type).html("");
+        $("#x_axis-" + type).html("");
+        $("#slider-" + type).html("");
+    }
+
+    function initGraph(type, isMultilined) {
+        if (graphMap[type]) {
+            return graphMap[type];
+        }
+
+        var chartWrapperElmId = "#connectedcup-div-chart";
+        var graphWidth = $(chartWrapperElmId).width() - 50;
+
+        var graphConfig = {
+            element: document.getElementById("chart-" + type),
+            width: graphWidth,
+            height: 400,
+            strokeWidth: 2,
+            renderer: 'line',
+            interpolation: "linear",
+            unstack: true,
+            stack: false,
+            xScale: d3.time.scale(),
+            padding: {top: 0.2, left: 0.02, right: 0.02, bottom: 0.2},
+            series: []
+        };
+
+        if (devices) {
+            for (var i = 0; i < devices.length; i++) {
+                graphConfig['series'].push(
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': devices[i].name
+                        });
+            }
+        } else {
+            if (isMultilined) {
+                graphConfig['series'].push(
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "x"
+                        },
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "y"
+                        },
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "z"
+                        }
+                );
+            } else {
+                graphConfig['series'].push(
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': $("#connectedcup-details").data("devicename")
+                        });
```

Comment:
Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings